bambi/models.py

```diff
@@ -615,14 +500,138 @@ def _add_y(self, variable, prior=None, family="gaussian", link=None, *args, **kw
         if prior is None:
             prior = self.family.prior
 
+        variable = vector.design_info.term_names[0]
+
         if self.family.name == "gaussian":
             prior.update(sigma=Prior("HalfStudentT", nu=4, sigma=self.clean_data[variable].std()))
 
-        data = kwargs.pop("data", self.clean_data[variable])
-        term = Term(variable, data, prior=prior, *args, **kwargs)
-        self.y = term
+        # Success event when family = 'bernoulli'
+        success_event = None
+        categorical = False
+
+        if event is not None:
+            if self.family.name != "bernoulli":
+                raise ValueError("Index notation only available for 'bernoulli' family")
+            # pass in new Y data that has 1 if y=event and 0 otherwise
+            success_event = event.group(1)
+            categorical = True
+            data = vector[:, vector.design_info.column_names.index(success_event)]
+            data = pd.DataFrame({event.group(3): data})
```

Comment:
What is group 3? Can a comment be added here for clarification?

Reply:
It is already written in `_add_fixed()` method. 

```python
    def _add_fixed(self, fixed, data, family, link, priors):
        if "~" in fixed:
            # check to see if formula is using the 'y[event] ~ x' syntax.
            # If so, chop it into groups:
            # 1 = 'y[event]', 2 = 'y', 3 = 'event', 4 = 'x'
            # If this syntax is not being used, event = None
            event = re.match(r"^((\S+)\[(\S+)\])\s*~(.*)$", fixed)
```
But I think an reminder there will be helpful. Thanks!!

Reply:
Oh I see. my apologies. Treat this comment as optional then